apps/data/lib/data/opportunities.ex

```diff
@@ -44,6 +45,9 @@ defmodule Data.Opportunities do
     |> Repo.update
   end
 
+  defp filters(query, filters),
+    do: Enum.reduce(filters, query, fn {:level, value}, query -> where(query, [o], field(o, :level) in ^value) end)
```

Comment:
I feel like this is more readable with parens for the `fn` args but dont necessarily feel strongly about it.

Reply:
I don't really like this.  You know me, I'm not a huge anon function guy.  I will probably reconsider this and refactor.